R/print.R

```diff
@@ -442,6 +442,61 @@ print.bbi_nmboot_summary <- function(x, .digits = 3, .nrow = 10, ...) {
   }
 }
 
+
+#' @describeIn print_bbi Prints the `NM-TRAN` evaluation of a `bbi_nonmem_model`
+#'  object
+#' @export
+print.nmtran_process <- function(x, ...){
+
+  is_valid_print <- function(.x) {
+    if (!is.null(.x)) {
+      length(.x) != 0
+    } else {
+      FALSE
+    }
+  }
+
+  heading <- cli_h1
+  subheading <- cli_h2
+  bullet_list <- cat_bullet
+
+  status <- x[["status"]]
+  if (x[["status_val"]] == 0) {
+    status <- col_green(status)
+  } else {
+    status <- col_red(status)
+  }
+
+  nm_version <- x[["nonmem_version"]]
+  if(is.null(nm_version)) nm_version <- "unknown"
+
+  heading(cli::col_magenta("NM-TRAN Status"))
+  subheading(status)
+
+  heading("Absolute Model Path")
+  bullet_list(x[[ABS_MOD_PATH]])
+
+  heading("NM-TRAN Specifications")
+  cli::cli_bullets(
+    c(
+      "*" = "NM-TRAN Executable: {.path {x[['nmtran_exe']]}}",
+      "*" = "nmtran_presort: {cli::col_cyan(!is.null(x[['nmtran_presort_exe']]))}",
+      "*" = "NONMEM Version: {.val {nm_version}}"
+    )
+  )
+
+  # Print run directory if it still exists (clean = FALSE)
+  if(fs::dir_exists(x[['run_dir']])){
+    cli::cli_bullets(c("*" = "Run Directory: {.path {x[['run_dir']]}}"))
+  }
+
+  if (is_valid_print(x[["output_lines"]])) {
+    heading('Standard Output')
```

Comment:
"Standard Output" is inaccurate. It's a mix of standard output and standard error, so I think the suggestion from my [original comment](https://github.com/metrumresearchgroup/bbr/pull/705#discussion_r1686981210) about this ("Output") would be better.